crates/gen/build.rs

```diff
@@ -1,36 +1,24 @@
-#[cfg(windows)]
 fn main() {
-    let mut source: ::std::path::PathBuf = ::std::env::var("CARGO_MANIFEST_DIR")
-        .expect("No `CARGO_MANIFEST_DIR` env var")
-        .into();
+    let path_file =
```

Comment:
We can probably just revert to using `include_bytes` and avoid all of this build trickery. That macro is slow but not that slow to warrant all the extra complexity. And hopefully it will eventually improve. 

https://github.com/rust-lang/rust/issues/65818

Reply:
Indeed, I don't recall it being noticeable but then I haven't (had a reason to) performed a profile to find out either way. Will revert to it as we don't need this build trickery (unless we'd like to support "scenario 3", though even that might be possible with `include_bytes` one way or another).

Reply:
@kennykerr 098ef4926b2db7aa4afdf033dc04048cd313918e dropped the `if !result.iter().any(|file| file.name.starts_with("Windows."))` before adding "builtin" winmds too, that should be fine because these files are added at the end and would anyway be visited last.

Reply:
The `if` is needed in the event that a project provides its own `.windows/winmd` folder with its own winmd files for the Windows API.

Reply:
Yeah, but it was removed at some point. Wrongly? I was under the assumption that winmds were linearly scanned for types, returning on the first match. In that sense a custom winmd would have precedence?

Reply:
I believe this check exists for several reasons, like
* to strongly discourage use of the reserved (by the OS) `Windows.*` namespace, and
* to prevent developer confusion. Folks may grab a bunch of (older) Windows metadata from the SDK (they are misused often) and plop them into the winmd folder creating a mess

Hackers™️ can override this behavior by renaming their winmd.

I don't have a strong opinion here.

Reply:
@riverar Yes it would definitely be odd to pick up builtin winmds if the user explicitly uses different ones. But then filenames are at the same time fragile to solely base this on.

Reply:
The removal was accidental. Projects may need to use newer/different metadata. For example, building the next version of Windows requires the new metadata that isn't yet publicly available. Other projects based on Windows, like Xbox, often have their own set of Windows APIs that may look a bit different. We also have various internal teams that need to provide their own metadata for various reason like provenance. An of course the hackers. Anyway, this is important. 😉

Reply:
Obviously I'll need to add a test for this.

Reply:
Sure, it's added back now.